src/FixedDataTableWidthHelper.js

```diff
@@ -22,6 +22,33 @@ function getTotalWidth(/*array*/ columns) /*number*/ {
   return totalWidth;
 }
 
+function getMaxVisibleColumns(/*array*/ columns, /*number*/ width) /*number*/ {
```

Comment:
This is where I calculate the max number of columns that will be visible at one time.
Runs in O(n)